examples/meta/generator/targets/cpp.json

```diff
@@ -120,7 +120,37 @@
             "get_string": "$object->get<std::string>($arguments)"
         },
         "StaticCall": "$typeName::$method($arguments)",
-        "GlobalCall": "$method($arguments)",
+        "GlobalCall": {
+            "Default": "$method($arguments)",
```

Comment:
we really need to think about having the base classes defined in one global place rather than re-stating them over and over again. @gf712 maybe we could have them in a header file, and then a python script can parse that and take them out for situations like this one here?

Reply:
Hmm, I guess so. For plugins I think shogun-core would need to have all base class definitions?

Reply:
Yep!